emerging_optimizers/orthogonalized_optimizers/muon.py

```diff
@@ -67,6 +67,7 @@ def __init__(
         use_nesterov: bool = False,
         weight_decay: float = 0.01,
         use_decoupled_weight_decay: bool = True,
+        use_independent_weight_decay: bool = False,
```

Comment:
Change: The variable name is getting too long. let's make them "use_decoupled_we", "use_independent_wd".

I don't think there are ambiguities around wd being short for weight decay.

Reply:
Megatron can update, there will be a lot of update on dev anyway.
Nothing will break as dependency was made on commit not head of branch